index.html

```diff
@@ -5300,8 +5300,9 @@ <h4>Plain HTML or Polyfill DOM Result of the MathML Quadratic Formula</h4>
 		<div class="role" id="menu">
 			<rdef>menu</rdef>
 			<div class="role-description">
-				<p>A type of <a>widget</a> that offers a list of choices to the user.</p>
-				<p>A menu is often a list of common actions or functions that the user can invoke. The <code>menu</code> <a>role</a> is appropriate when a list of menu items is presented in a manner similar to a menu on a desktop application.</p>
+				<p>A type of <a>widget</a> that offers a list of choices to the user. A menu is <em class="rfc2119">usually</em> presented vertically and <em class="rfc2119">usually</em> does not remain visible.
```

Comment:
usually is not one of the rfc2119 keywords. It should not have the class="rfc2119".